website/content/docs/connect/k8s/crds.mdx

```diff
@@ -33,23 +50,20 @@ Hang tight while we grab the latest from your chart repositories...
 Update Complete. ⎈Happy Helming!⎈
 ```
 
-Refer to [Install with Helm Chart](/consul/docs/deploy/server/k8s/helm) for further installation
-instructions.
-
-**Note**: Configuration entries require `connectInject` to be enabled, which is a default behavior in the official Helm Chart. If you disabled this setting, you must re-enable it to use CRDs.
+Refer to [Install with Helm Chart](/consul/docs/deploy/server/k8s/helm) for
+further installation instructions.
 
-## Upgrading An Existing Cluster to CRDs
```

Comment:
I deleted this section because the spreadsheet says this is deprecated. The content for this seems to have been removed elsewhere.